app/components/companion_windows_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="sul-embed-container companion-window-component" id='sul-embed-object' hidden data-controller="companion-window iiif-manifest-loader fullscreen <%= @stimulus_controller %>" data-iiif-manifest-loader-iiif-manifest-value="<%= iiif_v3_manifest_url %>" data-fullscreen-target="area" data-fullscreen-close-value="Exit full screen">
```

Comment:
Can you move the first part into the class file?  Just append it into `@stimulus_controller`.  Perhaps the second part could also be extracted into a method like `fullscreen_options`?

Reply:
I went ahead and did this along those lines. i wasn't able to figure out how to do fullscreen_options without having weird escaping that probably would have required html_safe to get it to render correctly